client/ionic/src/pages/Triage.tsx

```diff
@@ -13,7 +15,22 @@ interface TriageState {
 class Triage extends React.Component<{}, TriageState> {
```

Comment:
@advayDev1 mentioned his preference for functional components somewhere, if you are up for it you could include that change here.